receiver/sapnetweaverreceiver/metadata.yaml

```
@@ -30,102 +37,174 @@ metrics:
-    unit: "{sessions}"
+  sapnetweaver.session.count:
+    description: The amount of of sessions created.
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.web.count:
-    description: The number of Web Sessions.
-    unit: "{sessions}"
+  sapnetweaver.queue.count:
+    description: The queue length.
+    unit: "{entr}"
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.browser.count:
-    description: The number of Browser Sessions.
-    unit: "{sessions}"
+  sapnetweaver.queue_peak.count:
+    unit: "{entr}"
+    description: The peak queue length.
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.ejb.count:
-    description: The number of EJB Sessions.
-    unit: "{sessions}"
+  sapnetweaver.job.aborted:
+    description: The amount of aborted jobs.
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.icm_availability:
-    description: ICM Availability (color value from alert tree).
+  sapnetweaver.abap.update.errors:
+    description: The amount of ABAP errors in update.
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
     attributes: [control_state]
     enabled: true
-  sapnetweaver.host.spool_list.used:
-    description: Host Spool List Used.
+  sapnetweaver.response.duration:
+    description: The response time duration.
+    unit: "ms"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: [response_type]
+    enabled: true
+  sapnetweaver.request.count:
+    description: The amount of requests made.
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+    enabled: true
+  sapnetweaver.request.timeout.count:
+    description: The amount of timed out requests.
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+    enabled: true
+  sapnetweaver.connection.errors:
+    description: The amount of connection errors.
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+    enabled: true
+  sapnetweaver.cache.hits:
+    description: The cache hit percentage.
+    unit: "%"
+    gauge:
+      value_type: int
+      input_type: string
+    attributes: []
+    enabled: true
+  sapnetweaver.cache.evictions:
+    description: The number of evicted entries.
+    unit: "{entr}"
```

```suggestion
    unit: "{entries}"
```

Was `entr` the desired unit?

---

Yeah, it is the value received. It is an abbreviation though, so I can update to entries